src/metabase/api/table.clj

```diff
@@ -53,13 +53,6 @@
                    (map fix-schema))
           tables)))
 
-(api/defendpoint GET "/uploaded"
```

Comment:
Moved this to the new EE route context

Reply:
did we release this API? if so we need to add this to the API change log

Reply:
We didn't

Reply:
we're good then.

but for future references, if we remove/move API, we should mention it right?

Reply:
yes absolutely